lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerConnectTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Puppeteer
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class PuppeteerConnectTests : PuppeteerBaseTest
+    {
+        [Fact]
+        public async Task ShouldBeAbleToConnectMultipleTimesToSameBrowser()
+        {
+            var options = new ConnectOptions()
+            {
+                BrowserWSEndpoint = Browser.WebSocketEndpoint
+            };
+            var browser = await PuppeteerSharp.Puppeteer.ConnectAsync(options);
+            using (var page = await browser.NewPageAsync())
+            {
+                var response = await page.EvaluateExpressionAsync<int>("7 * 8");
+                Assert.Equal(response, 56);
+            }
+
+            using (var originalPage = await Browser.NewPageAsync())
+            {
+                var response = await originalPage.EvaluateExpressionAsync<int>("7 * 6");
+                Assert.Equal(response, 42);
+            }
+        }
+
+        [Fact]
+        public async Task ShouldBeAbleToReconnectToADisconnectedBrowser()
+        {
+            var options = new ConnectOptions()
+            {
+                BrowserWSEndpoint = Browser.WebSocketEndpoint
+            };
+
+            var url = TestConstants.CrossProcessHttpPrefix + "/frames/frame.html";
```

Comment:
Why aren't we testing it using `/frames/nested-frames.html` as puppeteer does?
I saw that you were doing that before